### DISCUSSION TITLE
Changing screenshots on app stores

### DISCUSSION THREAD

**Discussion by NikunjK8** [T8RIN/ImageToolbox#944] [Ideas]
Imo the screenshots of app on play store & f-droid doesn't do it justice. They look very dark, shadowed and there's a lot of empty space in frames. You could show different features in different colors (specially in light mode) to show how beautiful this app is. This might also help the app reach a wider audience. What do you think?

**Comment by T8RIN** [comment]
I used default pattern, there was old screenshots with light theme, but i made them dark for github dark theme

**Comment by T8RIN** [comment]
![image](https://github.com/T8RIN/ImageToolbox/assets/52178347/fad2a923-7856-474d-adb8-deccd18c9eed)
old is upper ones

**Reply by NikunjK8** [reply]
How about a mix of them?

**Reply by T8RIN** [reply]
How?🐸

**Reply by NikunjK8** [reply]
Keep Dark ones on Github and upload light ones on app stores with one or two screenshots showcasing dark mode support?

**Reply by T8RIN** [reply]
I think style should be the same within light or dark theme, so i will create light ones too

**Reply by NikunjK8** [reply]
Ok good point. Will close the discussion now
